### PR TITLE
RN: Update version to 3.1 (Dunfell)

### PR DESCRIPTION
**Pull request by angolini**
Signed-off-by: Daiane Angolini <daiane.angolini@nxp.com>

### CODE REVIEW

File: release-notes/source/linux-default.inc [hunk 1]
```diff
@@ -1,75 +1,84 @@
...
-tx6u-80x0               linux-karo           3.16-2015-09-18
-tx6u-81x0               linux-karo           3.16-2015-09-18
-ventana                 linux-gateworks-imx  3.14-1.0.x-ga+yocto
-wandboard               linux-wandboard      4.1-2.0.x
-======================  ===================  ==========================
+======================  ===================  ====================
+        Board             Kernel Provider       Kernel Version
+======================  ===================  ====================
+apalis-imx6             linux-fslc           5.4.58+git
+cgtqmx6                 linux-congatec       ${LINUX_VERSION}+git
```
Comment (otavio) [inline release-notes/source/linux-default.inc:5]: @angolini this is likely due `linux-congatec` do not have `LINUX_VERSION` defined. Can you take a look at it or wish us to look at it?

Comment (otavio) [reply release-notes/source/linux-default.inc:5]: Yes, please do.

Comment (angolini) [reply release-notes/source/linux-default.inc:5]: https://github.com/Freescale/meta-freescale-3rdparty/pull/92
https://github.com/Freescale/meta-freescale-3rdparty/pull/93

**Comment by otavio** [comment]
@angolini I merged it, can you update the branch here?

**Comment by angolini** [comment]
@otavio Do you mean the way I just did?

(only double checking I understand what is "here")

**Pull request merged**